### ISSUE TITLE
free dudo

### ISSUE THREAD

**Issue by EvelyneCalista** [Masaaki-75/freeseed#3]
Hi @Masaaki-75 ,

I tried the freedudo model,
I observed that the train and validation losses are very high >200. But the PSNR shows a good result like below.
Do i miss something in the parameters?
![image](https://github.com/Masaaki-75/freeseed/assets/71305979/c7e04f16-4c32-49ff-923e-c3ec594dec9e)

here is the inference result
![image](https://github.com/Masaaki-75/freeseed/assets/71305979/3cb21d78-fd1c-4d2b-a2bb-513b999b7e3a)

Thank you!

**Comment by Masaaki-75** [comment]
In most cases, you don't need to worry about the loss value as long as the PSNR (or other well-defined metrics) is good. The dudofree model has several loss terms, and the magnitude of loss on the sinogram domain is usually larger than that of loss on the image domain.
